microsoftgraph/microsoft-graph-explorer-v4
# Disable invalid space error on url on carriage returns

## Overview
Solves #1454 

### Demo
![image](https://user-images.githubusercontent.com/45680252/154433454-77a7f855-b21b-4809-875a-7917ae733d06.png)

![image](https://user-images.githubusercontent.com/45680252/154433561-1ddc020e-26ce-4860-a066-6b8a61dadf78.png)

## Testing Instructions
Type a long url on the url textbox. Press enter a couple of times. Notice that the error is not displayed
